db/seeds.rb

```diff
@@ -4,4 +4,22 @@
 # Examples:
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+require "nokogiri"
+require "open-uri"
+
+page = Nokogiri::HTML(open("http://www.homeenglish.ru/250Popular1.htm"))
+page.css('#middle > table tr:not(.first)').each do |item|
+  i=1
+  2.times do
+    orig = item.css('td')[i].text
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.